### ISSUE TITLE
Consecutive whitespace in query causes error in Custom Search

### ISSUE THREAD

**Issue by sh41** [getsentry/sentry#4940]
Version: 8.13.0
Installation: Python install

When searching in the custom search I get an internal error when there are consecutive white spaces in a quoted element: 
This says "Sorry, no events match your filters.": 
http://sentry.example.com/MYORG/MYPROJECT/?query=user%3A%22test+spaces%22

This says "Internal Error": 
http://sentry.example.com/MYORG/MYPROJECT/?query=user%3A%22test++spaces%22

Stacktrace:
```Traceback (most recent call last):
  File "/srv/sentry/virtualenv/local/lib/python2.7/site-packages/sentry/api/base.py", line 86, in handle_exception
    return super(Endpoint, self).handle_exception(exc)
  File "/srv/sentry/virtualenv/local/lib/python2.7/site-packages/sentry/api/base.py", line 180, in dispatch
    response = handler(request, *args, **kwargs)
  File "/srv/sentry/virtualenv/local/lib/python2.7/site-packages/sentry/api/endpoints/project_group_index.py", line 239, in get
    query_kwargs = self._build_query_params_from_request(request, project)
  File "/srv/sentry/virtualenv/local/lib/python2.7/site-packages/sentry/api/endpoints/project_group_index.py", line 149, in _build_query_params_from_request
    query_kwargs.update(parse_query(project, query, request.user))
  File "/srv/sentry/virtualenv/local/lib/python2.7/site-packages/sentry/search/utils.py", line 263, in parse_query
    tokens = tokenize_query(query)
  File "/srv/sentry/virtualenv/local/lib/python2.7/site-packages/sentry/search/utils.py", line 246, in tokenize_query
    while nvalue[-1] != '"':
IndexError: string index out of range
```

**Comment by MaxBittker** [comment]
Thanks for the reproducible example sh41

**Comment by travisoneill** [comment]
Opened pr #5100 to resolve

**Comment by mattrobenolt** [comment]
Should be fixed with #5100
